cards_corrections.json

```diff
@@ -1,2 +1,12 @@
 {
-}
\ No newline at end of file
+  "Angel of the God-Pharoh": {
+    "name": "Angel of the God-Pharaoh",
+    "URL": "http://mythicspoiler.com/hou/cards/angelofthegodpharaoh.jpg"
+  },
+  "Maurauding Boneslasher": {
+    "URL": "http://mythicspoiler.com/hou/cards/maraudingboneslasher.jpg"
+  },  
+  "Chandra's Defeat": {
+    "URL": "http://mythicspoiler.com/hou/cards/chandrasdefeat.jpg"
```

Comment:
Ah I hadn't realised the english art was out. Only mtgs has the new art - i'll change it to them.